phpBB/phpbb/avatar/manager.php

```diff
@@ -331,6 +339,19 @@ public function handle_avatar_delete(\phpbb\db\driver\driver_interface $db, \php
 				WHERE user_avatar = '" . $db->sql_escape($avatar_data['avatar']) . "'";
 			$db->sql_query($sql);
 		}
+
+		/**
+		* Event is triggered after user avatar has been deleted
+		*
+		* @event core.avatar_manager_avatar_delete_after
+		* @var	\phpbb\user	user		User to manupulate with
```

Comment:
I'd rather just say "phpBB user object". Also, it should be manipulate not manupulate.